pkgs/applications/science/logic/surelog/default.nix

```diff
@@ -7,20 +7,30 @@
 , libuuid
 , openjdk11
 , gperftools
+, flatbuffers
+, fetchpatch
 }:
 
 stdenv.mkDerivation rec {
   pname = "surelog";
-  version = "1.40";
+  version = "1.45";
 
   src = fetchFromGitHub {
     owner = "chipsalliance";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-5nhJilFIJJDCnJUEUgyPNtWSQUgWcvM6LDFgFatAl/k=";
+    hash = "sha256-/SSKcEIhmWDOKN4v3djWTwZ5/nQvR8ibflzSVFDt/rM=";
     fetchSubmodules = true;
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/chipsalliance/Surelog/commit/9a54efbd156becf65311a4272104810f36041fa6.patch";
+      sha256 = "sha256-rU1Z/0wlVTgnPLqTN/87n+gI1iJ+6k/+sunVVd0ulhQ=";
+      name = "parallel-test-running.patch";
+    })
+  ];
```

Comment:
Why do we need this? When can we remove this again?

A comment would be helpful.

Reply:
It prevents race conditions in tests. These show up as we run `ctest` with parallel execution enabled.

It can be removed with the next tagged version of this project.

Reply:
Added comment.